src/components/forms/OfficeHoursForm.tsx

```diff
@@ -84,15 +84,15 @@ export const OfficeHoursForm: FC = () => {
       pb={20}
     >
       <form id='office-hours-form' onSubmit={handleSubmit(onSubmit)}>
-        <FormControl id='first-name' isRequired mb={8}>
+        <FormControl id='first-name-control' isRequired mb={8}>
```

Comment:
Wondering, what do we use these ids for? its is something related with the validation?

Reply:
I've added them to avoid errors on compilation, otherwise NextJS displays warning messages

Reply:
Got it, thanks.